src/components/v5/shared/Modal/ModalBase.tsx

```diff
@@ -33,6 +34,7 @@ const ModalBase: FC<ModalBaseProps> = ({
         beforeClose: 'blur-none',
       }}
       className={clsx(
+        className,
```

Comment:
You also won't need this className if you use `hasPadding` instead, as @mmioana proposed 😊